src/_guides/language/sound-problems.md

```diff
@@ -607,8 +542,8 @@ Do we have any known issues or bugs to list here?
 ### The covariant keyword
 
 Some (rarely used) coding patterns rely on tightening a type
-by overriding a parameter's type with a subtype, which is illegal in strong
-mode Dart. In this case, you can use the `covariant` keyword to
+by overriding a parameter's type with a subtype, which is illegal.
+In this case, you can use the `covariant` keyword to
 tell the analyzer that you are doing this intentionally.
 This removes the static error and instead checks for an invalid
 parameter type at runtime.
```

Comment:
"parameter type" -> "argument type"
(We generally use "parameter" about the formal parameters declared by a function and "argument" about actual parameters passed to the function by a function call).